wiki/FAQ/en.md

```diff
@@ -1,731 +1,99 @@
-
-First, tap on it at the right moment, however, it does not end there.
-A ball will begin to move across the path.
-Hold the mouse/keyboard-button (or keep the pen onto the tablet) and follow the ball within the thick orange circle, surrounding the ball, as it moves.
-The orange outer circle, called the follow circle, will appear if when holding onto the slider's ball, but will disappear when released.
-
-Sometimes, as seen in the screenshot above, the ball may reverse its direction and the player must follow the ball back to the start of the path or vice versa.
-The visual cue is a reverse arrow at the ending/starting circle of the path.
-
-##### Spinners
-![Spinner][osu_spinner image]
-
-_A cleared spinner with spinner bonus_
-
-Hold on the mouse/keyboard-button (or keep the pen onto the tablet).
-From there, use the mouse (or pen) and spin the spinner in a circular motion (in either direction) until the spinner circle grows outwards completely.
-A _Clear_ notice will appear to indicate that the spinner was completed.
-If the spinner was cleared early, continue spinning to collect score bonus and gain some health back.
+Got any questions or doubts? If yes, we are here to help you! Below we've prepared a set of answers for the **frequently asked questions**
 
-The outer white circle shows how much time left to complete the spinner.
-This circle will turn red to notify that time is almost out.
-Older beatmaps, using _skin version 1.0_, will have a meter/gauge to indicate how close to clearing the spinner.
+## osu!
 
-The small box below the spinner shows spins per minute.
-The number inside that box translates to the number of spins possible, if to continue to spin the spinner at that speed for one minute.
-If mouse/keyboard-button was released (or pen did not touch the tablet) early during the spin, the spinner will stop reading the spin.
+### What is that "Quendan" thing?
 
-On the _DS_, this is a good way to scratch the touchscreen (or screen protector), especially on the higher difficulties.
-In osu!, spinners have been toned down as to not be such a nuisance to complete.
-The maximum number of spins per minute achievable is 477 which is also the rate at which the [Auto][Auto wikilink] game modifier spins while the [Spun Out][Spun Out wikilink] game modifier spins slower at 287 spins per minute.
+### How many Game Modes are there?
 
-#### In-Game Interface
-![osu! interface][osu! Interface image]
+### What are those Game Modifications?
 
-_In-game interface of osu!_
+### Where to find and get the beatmaps?
 
-The top-left bar is the health bar, which will decrease at a steady rate (depending on the beatmap's difficulty settings), but can be replenished by tapping notes at the right time or spinning the spinner.
-A perfectly timed hit (a 300 or Geki) will recover health more than a badly timed hit (50).
-A total miss will take a good chunk of health out of the health bar.
+### How does the Grading system work?
```

Comment:
ditto; what letter casing are you using?
title case -> `How Does the Grading System Work?`
sentence case -> `How does the grading system work?`